Added labeller function and updated dependencies

the-Hull opened the pull request:
Dear devs,

thanks for the effort you've put into this package / data set. 
In trying to re-build the package with a unit labelling function, I noticed issues with tests (`storr::hash_object()`) causing my builds to fail. 

I updated the dependencies in (including a md5 hash) in the tests, and added the labeller function (with documentation and corresponding tests) - perhaps somebody will find it useful.

Cheers and stay healthy,
Alex

the-Hull committed:
Create test-baad-labeller

the-Hull committed:
Update test-baad-labeller

the-Hull committed:
Create baad_labeller

the-Hull committed:
Updated filename

the-Hull committed:
updated tests

the-Hull committed:
# This is a combination of 3 commits.
# This is the 1st commit message:

updated tests

# This is the commit message #2:

update travis.yml

# This is the commit message #3:

new travis yml

the-Hull committed:
update travis.yml

the-Hull committed:
new travis yml

the-Hull committed:
updated travis

the-Hull committed:
fix travis

the-Hull committed:
fix travis

the-Hull committed:
updated hash val

the-Hull committed:
update travis

dfalster commented:
Hi @the-Hull ! Thanks heaps for this. Really appreciate your input and suggestions for new features. I'll be interested to see the labeller function.  I'll check this out in detail hopefully next week. But just wanted to say a quick thank you in the meantime.

the-Hull commented:
No worries! Quick example:

```
libary(baad.data)

bd <- baad_data()

xlab <- baad_labeller(bd$dictionary, "r.st", as_expression = TRUE)
ylab <- baad_labeller(bd$dictionary, "h.t", as_expression = TRUE)

plot(bd$data$r.st, bd$data$h.t, xlab = xlab, ylab = ylab)
```

the-Hull committed:
labeller, adjusted internal check

dfalster commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # baad.data
 
-[![Build Status](https://travis-ci.org/traitecoevo/baad.data.png?branch=master)](https://travis-ci.org/traitecoevo/baad.data)
+[![Build Status](https://travis-ci.org/the-Hull/baad.data.svg?branch=master)](https://travis-ci.org/the-Hull/baad.data)
```
Can we change this back to traitecoevo?

the-Hull replied:
Sorry for the late follow up @dfalster 
I'll get to this this week, most likely! 
Cheers!

the-Hull replied:
Sorry for the delay - I will try to squash the commits!

dfalster reviewed (changes requested):
Thanks for the updates @the-Hull. These additions are very helpful and I'd like to merge them in.

Can I request two changes first?

1. Fix travis link to be traitecoevo
2. Update the pull request to include 1-2 commits only? currently there's 13 commits - most of which are just intermediaries. Either I merge squash your work into a single commit, or you update the PR to include only 1-2 commits and I merge that without squashing.

the-Hull committed:
# This is a combination of 8 commits.
# This is the 1st commit message:

updated tests

# This is the commit message #2:

update travis.yml

# This is the commit message #3:

new travis yml

# This is the commit message #4:

updated travis

# This is the commit message #5:

fix travis

# This is the commit message #6:

fix travis

# This is the commit message #7:

updated hash val

# This is the commit message #8:

update travis

the-Hull committed:
fixed labeller

the-Hull committed:
Merge branch 'master' of https://github.com/the-Hull/baad.data

the-Hull committed:
updated readme
